<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>971</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chat 28 Admin Console</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="Server_Label">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>Server:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="Server_lineEdit">
       <property name="text">
        <string>localhost</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Connect_pushButton">
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="BanpushButton">
      <property name="minimumSize">
       <size>
        <width>953</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Ban/Unban selected users</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTableWidget" name="Users_tableWidget">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <column>
        <property name="text">
         <string>ID</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Login</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Name</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>email</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Hash</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Banned till</string>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="foreground">
         <brush brushstyle="NoBrush">
          <color alpha="255">
           <red>170</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </property>
       </column>
      </widget>
      <widget class="QTableWidget" name="Messages_tableWidget">
       <column>
        <property name="text">
         <string>Time</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>From</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>To</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Body</string>
        </property>
        <property name="textAlignment">
         <set>AlignLeading|AlignTop</set>
        </property>
       </column>
       <column>
        <property name="text">
         <string>ID</string>
        </property>
       </column>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>971</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuExit">
    <property name="title">
     <string>Main</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionLaunch_another_session"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuExit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionNew_Client_Session">
   <property name="text">
    <string>New Client Session</string>
   </property>
  </action>
  <action name="actionLaunch_another_session">
   <property name="text">
    <string>Launch another session</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
